### DISCUSSION TITLE
Adding New Theme

### DISCUSSION THREAD

**Discussion by Mr-Bangladesh** [cpeditor/cpeditor#987] [Q&A] [answered]
Hello!! It's a great IDE!!! But is there any way to add new theme?

**Comment by ouuan** [comment, marked as answer]
[More themes](https://github.com/KDE/syntax-highlighting/tree/master/data/themes) will be available after #516 is done. Unfortunately, there's no ETA.

**Reply by funcdfs** [reply]
feat:

There is no simple black theme similar to `atom dark`, no italics and bold

add a css import entrance ~~or a plugin system~~？
